src/runtime/audience-action-flow.ts

```diff
@@ -333,7 +354,10 @@ export class AudienceActionIframeFlow implements AudienceActionFlow {
     const {onResult} = this.params_;
     if (onResult) {
       try {
-        return await onResult(request);
+        return await onResult({
+          configurationId: this.params_.configurationId,
+          data: request,
```

Comment:
What does it mean to return the request?

Reply:
The result is used in handleSurveyDataTransferRequest_ as:
```
const dataTransferSuccess = await this.attemptSurveyDataTransfer(request);
```
The result of the callback is used to determine if the data has been stored correctly.